.jenkins/pytorch/build.sh

```diff
@@ -31,7 +31,7 @@ if [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda9*gcc7* ]] || [[ "$BUILD_ENVIRONMENT"
 fi
 
 if [[ "$BUILD_ENVIRONMENT" == *pytorch-linux-xenial-py3-clang5-asan* ]]; then
-  exec "$(dirname "${BASH_SOURCE[0]}")/build-asan.sh" $*
+  exec "$(dirname "${BASH_SOURCE[0]}")/build-asan.sh" "$*"
```

Comment:
Is this always correct? Don't we want to re-interpret the arguments sometimes? This passes now cause the arguments happen to be empty.

Reply:
Hmm... aside from the fact that I made an incorrect replacement here (`shellcheck` actually recommended replacing `$*` with `"$@"`; see [SC2048](https://github.com/koalaman/shellcheck/wiki/SC2048)), perhaps this is a legit use case for `$*`, and it should be ignored like so:

    # shellcheck disable=SC2048

Reply:
`"$@"` doesn't mean what you think it does; it ensures that the constituent arguments of $@ are individually quoted correctly; c.f. https://stackoverflow.com/questions/3898665/what-is-in-bash

You'd better had a very good reason to drop the quotes.